api.go

```diff
@@ -68,24 +74,15 @@ func newNetAPI(root string, key string, proxy string) (*netAPI, error) {
 
 	q := u.Query()
 	q.Set("key", key)
-	q.Set("alt", "proto")
 	u.RawQuery = q.Encode()
 	return &netAPI{url: u, client: httpClient}, nil
 }
 
-// doRequests performs a POST to requestPath. It uses the marshaled form of req
-// as the request body payload, and automatically unmarshals the response body
-// payload as resp.
-func (a *netAPI) doRequest(ctx context.Context, requestPath string, req proto.Message, resp proto.Message) error {
-	p, err := proto.Marshal(req)
-	if err != nil {
-		return err
-	}
-
-	u := *a.url // Make a copy of URL
-	u.Path = requestPath
-	httpReq, err := http.NewRequest("POST", u.String(), bytes.NewReader(p))
-	httpReq.Header.Add("Content-Type", "application/x-protobuf")
+// doRequests performs a GET to requestPath. It automatically unmarshals the
+// response body payload as resp.
+func (a *netAPI) doRequest(ctx context.Context, urlString string, resp proto.Message) error {
+	httpReq, err := http.NewRequest("GET", urlString, nil)
+	httpReq.Header.Add("Content-Type", "application/json")
 	httpReq.Header.Add("User-Agent", "Webrisk-Client/0.1.2")
```

Comment:
Please update the version. I think 0.1.3 should be fine.

Reply:
Done, thanks!